### PR TITLE
Added condition for Discussions link

### PR DESCRIPTION
**Pull request by jayaike**
I have added the condition to display the Discussions Link

### CODE REVIEW

**Comment by jayaike** [comment]
@Sanji515 @vkartik97 Please Take a look

File: src/app/components/challenge/challenge.component.html [hunk 1]
```diff
@@ -125,7 +125,7 @@
                                     [routerLink]="['./leaderboard']"
                                     class="text-light-black fw-semibold fs-15">
                                     <i class="fa fa-th-list"></i> Leaderboard</a></li>
-                            <li *ngIf=""><a
+                            <li *ngIf="challenge.forumURL"><a
```
Comment (Sanji515) [inline src/app/components/challenge/challenge.component.html]: It will be better to add condition for `enable forum` field also.

Comment (jayaike) [reply src/app/components/challenge/challenge.component.html]: I don't understand @Sanji515

Comment (jayaike) [reply src/app/components/challenge/challenge.component.html]: I get it now @Sanji515 ....Done!!

File: src/app/components/challenge/challenge.component.html [hunk 2]
```diff
@@ -125,7 +125,7 @@
                                     [routerLink]="['./leaderboard']"
                                     class="text-light-black fw-semibold fs-15">
                                     <i class="fa fa-th-list"></i> Leaderboard</a></li>
-                            <li *ngIf=""><a
+                            <li *ngIf="challenge.forumURL && challenge.enable_forum"><a
```
Comment (Sanji515) [inline src/app/components/challenge/challenge.component.html]: Hey, I think it should be `challenge.forum_url` instead of `challenge.forumURL`.

Comment (jayaike) [reply src/app/components/challenge/challenge.component.html]: Done!! @Sanji515

**Comment by jayaike** [comment]
I made all the requested changes @Sanji515 @vkartik97 .. PTAL :) ⭐

**Review by lunayach — APPROVED** [review]
It's working. LGTM :tada:

**Review by Sanji515 — APPROVED** [review]
Thanks for the changes @nsjcorps

**Comment by jayaike** [comment]
Can I please get the task approved @Sanji515  😃

**Comment by Sanji515** [comment]
> Can I please get the task approved @Sanji515

yup, Done!

**Comment by jayaike** [comment]
Thanks a mill @Sanji515

**Comment by Sanji515** [comment]
@nsjcorps Can you please update the branch here?

**Pull request merged**